src/SpectatorView.Native/SpectatorView.Compositor/Compositor/AzureKinectFrameProvider.h

```diff
@@ -25,15 +26,24 @@ class AzureKinectFrameProvider : public IFrameProvider
         return _captureFrameIndex;
     }
 
+    uint8_t* GetBodyIndexBuffer(k4a_capture_t capture);
+    void UpdateSRV(k4a_image_t bodyDepthImage, ID3D11ShaderResourceView* _srv);
+    void SetBodyDepthBuffer(uint16_t* bodyDepthBuffer, uint16_t* depthBuffer, uint8_t* bodyIndexBuffer, int bufferSize);
```

Comment:
These can probably be in the private: block unless they need to be called from outside the class